INSTALL

```diff
@@ -48,6 +45,12 @@ WHAT DO YOU NEED ?
    - for Coqide, the Lablgtk development files, and the GTK libraries
      incuding gtksourceview, see INSTALL.ide for more details
 
+   Opam (https://opam.ocaml.org/) is recommended to install ocaml and
+   the corresponding packages.
+
+   $ opam install ocamlfind camlp5 lablgtk-extras
+
+   should get you a reasonable Ocaml enviroment to compile Coq.
```

Comment:
The `lablgtk-extras` is for CoqIDE, right? So it's not strictly needed. Should that be mentioned?

Reply:
It is mentioned just above.

Reply:
Yes, but it's not mentioned that building CoqIDE is optional, so installing `lablgtk-extras` is optional.

Reply:
@ejgallego can you make it more precise?

Reply:
I think everybody is busy so I'll merge as it is. @psteckler feel free to open a new PR to make it even better.